corehq/ex-submodules/casexml/apps/phone/restore.py

```diff
@@ -779,6 +804,24 @@ def _record_timing(self, status):
 
         datadog_counter('commcare.restores.count', tags=tags)
 
+    def __repr__(self):
+        return \
+            "RestoreConfig(project='{}', domain={}, restore_user={}, params='{}', cache_settings='{}', " \
+            "is_async='{}', restore_state='{}', force_cache='{}', cache_timeout='{}', overwrite_cache='{}', " \
+            "timing_context='{}')".format(
```

Comment:
nit: pep-8 recommends to use parentheses rather than line continuation characters (`\`). Python automatically concatenates strings across lines, so this might be reformatted something like:
```py
return (
    "RestoreConfig("
    "project={rc.project!r}, "
    "domain={rc.domain!r}, "
    "restore_user={rc.restore_user!r}, "
    "params={rc.params!r}, "
    "cache_settings={rc.cache_settings!r}, "
    "is_async={rc.is_async!r}, "
    "restore_state={rc.restore_state!r}, "
    "force_cache={rc.force_cache!r}, "
    "cache_timeout={rc.cache_timeout!r}, "
    "overwrite_cache={rc.overwrite_cache!r}, "
    "timing_context={timing}"
    ")"
).format(rc=self, timing=timing_context.to_dict())
```